### PR TITLE
[DEVOPS-493] csl-daedalus `applicationVersion`:  bump to 5

### PR DESCRIPTION
**Pull request by deepfire**

### CODE REVIEW

**Comment by gromakovsky** [comment]
Why 5? Do I recall correctly that mainnet is running 2 and staging is running 3?

**Comment by deepfire** [comment]
@gromakovsky, `rc-staging` runs `4`: https://github.com/input-output-hk/internal-documentation/wiki/Daedalus-installer-history

File: node/configuration.yaml [hunk 1]
```diff
@@ -14817,7 +14817,7 @@ mainnet_wallet_macos64: &mainnet_wallet_macos64
   update:
     <<:
     applicationName: csl-daedalus
-    applicationVersion: 3
+    applicationVersion: 5
```
Comment (gromakovsky) [inline node/configuration.yaml]: Why did you change it here and on line 14808? It should be changed only for staging (i. e. `mainnet_dryrun…` configurations), no?

Comment (deepfire) [reply node/configuration.yaml]: @gromakovsky, good catch!  Thank you!

**Comment by deepfire** [comment]
@domenkozar, this would dismiss review. It might be wise not to delay this any further.

**Comment by domenkozar** [comment]
If all PRs can have issue attached, this one should as well.
